content/en/_index.md

```diff
@@ -197,6 +231,38 @@ public static void listInfoTypes() throws IOException {
   }
 }
 {{< /tab >}}
+{{< tab header="PHP" >}}
+/**
+ * Lists all Info Types for the Data Loss Prevention (DLP) API.
+ */
+use Google\Cloud\Dlp\V2\DlpServiceClient;
+
+/** Uncomment and populate these variables in your code */
```

Comment:
This is the old-style for PHP samples, and should not be recommended

Reply:
@bshaffer is updating the DLP samples to the new PHP style in https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1615. Once merged I'll replace with the new style and we can then discuss if it's aligned with the guide.

Reply:
Done!